src/main/kotlin/com/cognifide/gradle/aem/common/AemExtension.kt

```diff
@@ -119,6 +121,11 @@ open class AemExtension(@Internal val project: Project) {
     val instances: List<Instance>
         get() = filterInstances()
 
+    @Internal
+    val environmentOptions = EnvironmentOptions()
+
+    fun environment(configurer: EnvironmentOptions.() -> Unit) = environmentOptions.apply(configurer)
```

Comment:
that's bad. return type will be `EnvironmentOptions` ; please convert to method body

Reply:
OK